storage/replica.go

```diff
@@ -1142,36 +1143,37 @@ func (r *Replica) handleRaftReady() error {
 			if err := command.Unmarshal(ctx.Payload); err != nil {
 				return err
 			}
-			if err := r.processRaftCommand(cmdIDKey(ctx.CommandID), e.Index, command); err == nil {
-				// TODO(bdarnell): update coalesced heartbeat mapping
-				r.mu.Lock()
-				r.mu.raftGroup.ApplyConfChange(cc)
-				r.mu.Unlock()
-			} else {
+			if err := r.processRaftCommand(cmdIDKey(ctx.CommandID), e.Index, command); err != nil {
```

Comment:
did this condition change? it used to check for `err == nil`

also, this error is completely discarded?